client/src/components/App/Content/LandingPage/InvestigationTable/InvestigatorAllocation/InvestigatorAllocationDialog.tsx

```diff
@@ -13,14 +13,20 @@ const title = 'הקצאת חקירה';
 const unSelectedRow = -1;
 
 const InvestigatorAllocationDialog: React.FC<Props> = (props) => {
-
-    const { isOpen, handleCloseDialog, investigators, allocateInvestigationToInvestigator, groupIds, epidemiologyNumbers } = props;
+    
+    let { isOpen, handleCloseDialog, investigators, allocateInvestigationToInvestigator, groupIds, epidemiologyNumbers } = props;
```

Comment:
the props need to be const its a bad practice to change them, if you need to change the value of prop clone his value to a new variable and change the new one 
read this article to learn more about why immutability is important [here](https://www.geeksforgeeks.org/why-is-immutability-so-important-in-javascript/)